src/zope/interface/common/__init__.py

```diff
@@ -259,5 +259,5 @@ def getRegisteredConformers(self):
         return set(itertools.chain(registered, self.__extra_classes))
 
 
-ABCInterface = ABCInterfaceClass.__new__(ABCInterfaceClass, 'ABCInterfaceClass', (), {})
+ABCInterface = ABCInterfaceClass.__new__(ABCInterfaceClass, 'ABCInterface', (), {})
```

Comment:
I think the `bases` argument should also match between `__new__` and `__init__`, although (given the previous None, None, None) I believe `__new__` ignores all three arguments.

Reply:
>  I believe __new__ ignores all three arguments.

That's essentially true, yes. But I can open a cleanup PR right quick.